src/tests/JIT/Methodical/MDArray/InnerProd/classarr.cs

```diff
@@ -22,6 +22,7 @@ public class intmm
 {
     public static int size;
     public static Random rand;
+    public const int DEFAULT_SEED = 20010415;
```

Comment:
Nit: here and in other places - I think the general style is to have the constants at the top, `PascalCase`d, separated from fields, so here it'd be:
```CS
public const int DefaultSeed = 20010415;

public static int size;
public static Random rand;
```